### PR TITLE
fix #9014 feat(nimbus): Always serialize feature configs under "features"

### LINKED ISSUE #9014: Remove feature key in V6 API
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3610)

### PR DESCRIPTION
**Pull request by brennie**
Because:
- the minimum version of Firefox for experiments is now 96; and
- multi-feature experiments are supported on all applications on version
  96+

this commit:
- unconditionally serializes feature configurations under the "features"
  field instead of "feature" in recipes; and
- serializes a "feature" field in recipes with a tombstone feature ID
  for mobile experiments like we do for desktop to prevent Firefoxen from
  crashing because they expected a "feature" field in the recipe.

### CODE REVIEW

**Comment by brennie** [comment]
Blocked on #9016

**Comment by jaredlockhart** [comment]
Yep and I talked to QA this morning and they'll test the mobile clients with `features` instead of `feature` then we can land this 👍

**Comment by brennie** [comment]
Blocked on QA ([QA-1977](https://mozilla-hub.atlassian.net/browse/QA-1977))

**Review by jaredlockhart — APPROVED** [review]
@brennie QA has finished testing this on desktop/mobile 96-116 and everything passed with flying colours so we can unblock this now 🎉 🎉 🎉 🎉 🎉

**Pull request merged**
